src/core/qgsdistancearea.h

```diff
@@ -304,6 +319,25 @@ class CORE_EXPORT QgsDistanceArea
     double computeDistanceBearing( const QgsPoint &p1, const QgsPoint &p2,
                                    double *course1 = nullptr, double *course2 = nullptr ) const;
 
+    /**
+     * Given a location, an azimuth and a distance, computes the
+     * location of the projected point. Based on Vincenty's formula
+     * for the geodetic direct problem as described in "Geocentric
+     * Datum of Australia Technical Manual", Chapter 4. Tested against:
```

Comment:
That would be the removal of the original source.

The whole text if from the original function
* with the source added